wolfhsm/wh_comm.h

```diff
@@ -58,14 +53,26 @@ enum {
 /* Header for a packet, request or response. On-the-wire format */
 typedef struct {
     uint16_t magic;     /* Endian marker with version */
-    uint16_t type;      /* Type of packet.  Enumerated in message.h */
+    uint16_t kind;      /* Kind of packet.  Enumerated in message.h */
     uint16_t seq;       /* Sequence number. Incremented on request, copied for
                          * response. */
     uint16_t aux;       /* Session identifier for request or error indicator
                          * for response. */
-} whHeader;
+} whCommHeader;
 /* static_assert(sizeof_whHeader == WOLFHSM_COMM_HEADER_LEN,
-                 "Size of whHeader doesn't match WOLFHSM_COMM_HEADER_LEN") */
+                 "Size of whCommHeader doesn't match WOLFHSM_COMM_HEADER_LEN") */
```

Comment:
I'm a big fan of `static_assert`, can we add something like this back in?

Reply:
Agreed.  _Static_assert() doesn't get in until C11 and so far I'm only using C99.  I think there's an equivalent in assert.h for GNU builds.  I'll take a peek.